torchtune/modules/low_precision/_register_nf4_dispatch_ops.py

```diff
@@ -17,26 +17,3 @@ def clone(func, *args, **kwargs):
     in precision.
     """
     return to_nf4(args[0][0].get_original_weight())
-
-
```

Comment:
starting from TorchAO==0.2.0, we implemented `NF.copy_`. It's the superset of `inplace_copy`. We cover both `NF4.copy_(bf16)` and `NF4.copy_(NF4)`